docs/guides/faq.md

```diff
@@ -0,0 +1,91 @@
+>  * Polyfills and unifies the native video api across browsers
+>  * Offers an extendable and themable UI
+>  * Takes care of accessibility for the user (in-progress)
+
+**Q: How do I install video.js?**
+> Currently video.js can be installed using bower, npm, or by serving a release file
+> from a github tag. For information on doing any of those see the [install guide](http://videojs.com/getting-started/).
+
+**Q: What do video.js version numbers mean?**
+> video.js follows [semver](http://semver.org/)
+
+**Q: How can I troubleshoot playback issues?**
+> See the [troubleshooting guide](/docs/guides/troubleshooting.md)
+
+**Q: What media formats does video.js support?**
+> This depends on the browsers support, video.js will work off of that. If Flash is avialable Flash videos
+> can also be played but only if the Flash tech is included with video.js.
+
+**Q: How can I hide the links to my video?**
+> This is outside of the scope of video.js and there are things that can be done to make it harder, but it will
+> never be completely possible.
+
+**Q: I think I found a bug with video.js. What should I do?**
+> If you know how to code and feel comfortable looking into the issue, we will
+> gladly review a pull request if you open one on the [video.js repo](https://github.com/videojs/video.js/pulls).
+> If you don’t have any idea what that means, please open an [issue on the video.js repo](https://github.com/videojs/video.js/issues)
+> and make sure that you follow the issue template so that we can better assist you.
+> If the bug relates to a plugin, please file the issue against the plugin’s repository.
+
+**Q: What is a [reduced test case](https://css-tricks.com/reduced-test-cases/)?**
+> A [reduced test case](https://css-tricks.com/reduced-test-cases/) is an example of the problem that you
+> are facing in isolation. Basically it is an example page that reproduces the issue in the least amount
+> of possible code. We have a [starter example](http://jsbin.com/axedog/edit?html,output)
+
+**Q: How do a make a plugin for video.js?**
+> See the [plugin guide](/docs/guides/plugins.md) for information on making a plugin for video.js
+
+**Q: How do I add a button to video.js?**
+> See the [button guide](/docs/guides/button.md) for information on adding a button to video.js
+
+**Q: Where can I find a list of video.js plugins?**
+> A list of plugins is avialable on [videojs.com](http://videojs.com/plugins).
+
+**Q: How can I get my plugin listend on the website?**
+> Add the 'videojs-plugin' keyword to your package.json and publish your package to npm.
+
+**Q: Where can I find a list of video.js skins?**
+> See the [video.js github wiki](https://github.com/videojs/video.js/wiki/Skins).
+
+**Q: Does video.js work as an audio only player?**
+> Yes! Video.js can be used with an `<audio>` tag the same way that it is used with a `<video>` tag.
```

Comment:
Not sure if we need to add it here but the spec (and videojs's different between the two going forward) is that the only difference between the two is that with `video` you have a display area and with `audio`, you do not.
You can play audio files in a video element and you can play a video file in an audio element.